content/en/security_platform/explorer.md

```diff
@@ -63,7 +63,7 @@ Below the header of Cloud Security Posture Management signals are tabs with deta
 
 Datadog Cloud SIEM offers threat intelligence feeds curated by threat intelligence partners. These feeds are constantly updated to include data about known suspicious activity (for example, indicators of compromise or IOCs), so you can quickly identify which potential threats to address.
 
-{{< img src="security_platform/security_monitoring/explorer/threat_intelligence.png" alt="Threat Intelligence in the Security Signals Explorer"  >}}
+{{< img src="security_platform/security_monitoring/explorer/threat_intelligence.png" alt="Threat Intelligence in the Security Signals Explorer" >}}
```

Comment:
This image has a (fake-looking) AWS account number, an IP, and an employee real name. It's also much bigger than strictly necessary to show the threat intel feature. Maybe consider editing it down, or blurring?